src/shims-plugins.d.ts

```diff
@@ -3,7 +3,7 @@ import { Gtm } from '@/plugins/google-tag-manager';
 import { Viewport } from '@/plugins/viewport';
 import { VueBemFunction } from '@/plugins/vue-bem-cn/src/globals';
 
-declare module '@vue/runtime-core' {
+declare module 'vue' {
```

Comment:
Attention:

export {}; need to be placed before declare in order to provide augmentation: 
https://vuejs.org/guide/typescript/options-api#type-augmentation-placement

also this change will lead to ts issues with eg: `this.$t`

to fix this: 

you will need to place the shims in your tsconfig.ts file

```
    "types": ["node", "vite/client", "src/shims-plugins.d.ts", "src/shims-vue.d.ts"],
```

with this everything should work as before